src/components/Swipeable.tsx

```diff
@@ -481,7 +499,7 @@ export default class Swipeable extends Component<
 
     return (
       <PanGestureHandler
-        activeOffsetX={[-10, 10]}
+        activeOffsetX={[-dragOffsetRight, dragOffsetLeft]}
```

Comment:
why right and then left? does "left" mean left edge of the screen or swiping to the left from the right edge of the screen? can we rename the props to make the names more explicit?

Reply:
In this case `dragOffsetRight` means the amount of drag right-to-left required for the gesture to start, and `dragOffsetLeft` means the amount of drag left-to-right.

The `right` is first, as `activeOffsetX` specifies a dead zone on the left and on the right. Since `dragOffsetRight` means dragging RTL, the dead zone needs to be on the left.

Would `dragOffsetRTL` and `dragOffsetLTR` be better (or non-abbreviated `dragOffsetRightToLeft` and `dragOffsetLeftToRight`)?

Reply:
`RTL` and `LTR` are strongly associated with the direction of the text, what about `dragOffsetFromLeftEdge` and `dragOffsetFromRightEdge`?